### ISSUE TITLE
Bug: cart item and payment mistmatch it's approved as order.

### ISSUE THREAD

**Issue by devenpateldp** [razorpay/razorpay-opencart#78]
- add to cart 1 item => go to checkout => select razorpay pay => click next => stand on razorpay page 
- open new tab on same browser - open same website and add more item to cart 
- now go back to payment page and complete your payment.
- order success with additional item and payment will be taken only for initially added to cart.

**Comment by manikumaran1978** [comment]
yes, this is serious bug. Many time the order value and the amount paid at razorpay is different. some time more than the order and some time less than the order total.
We did not face this problem in paytm in the last three years but in the last one month we started using razorpay and this problem is being faced

**Comment by abdulwahidsharief** [comment]
@devenpateldp for the above scenario two orders are created at razorpay payment for 1st order payment will be successful as opencart order is updated with higher amount callback will fail with error : `Invalid signature passed Payment Failed! Check Razorpay dashboard for details of Payment Id`

<img width="1439" alt="Image" src="https://github.com/user-attachments/assets/94939e02-42dd-46a8-8b25-9aa455ff007e" />
